### ISSUE TITLE
ExecEngine error messages don't show up as expected

### ISSUE THREAD

**Issue by RieksJ** [AmpersandTarski/prototype#70]
In default ExecEngine functions such as InsPair, DelPair, etc., there are explicit statemetns that count the number of arguments and raise an exception of which the text mentions the function that raised the error, as follows:

~~~php
ExecEngine::registerFunction('InsPair', function ($relationName, $srcConceptName, $srcAtom, $tgtConceptName, $tgtAtom) {
    /** @var \Ampersand\Rule\ExecEngine $this */
    if (func_num_args() != 5) {
        throw new Exception("InsPair() expects 5 arguments, but you have provided ".func_num_args(), 500);
    }
~~~

However, in practice, this is NOT what is presented:

![afbeelding](https://user-images.githubusercontent.com/8522589/57061057-da613a80-6cbb-11e9-9e79-105bc8368975.png)

This issue calls for the reinstatement of errors as they are geenrated from the PHP, so that the error stuff that is show to the developer makes sense and prevents him/her from tedious searching for what went wrong.

**Comment by Michiel-s** [comment]
I'm now working on a fix for this issue. See below how it will be.

The error message contains:
1. The place in the ADL code where a fix is needed
2. Explanation of what went wrong 

![image](https://user-images.githubusercontent.com/7961582/147603024-d4b6c40f-e63c-4049-a86b-67e461da67ff.png)

**Comment by Michiel-s** [comment]
Support for multiple tgt atoms in DelPair was already included in: fdb5849bfb34e605632f8dae2c08c09b152c6395
